controllers/numaresourcesoperator_controller.go

```diff
@@ -590,6 +592,57 @@ func (r *NUMAResourcesOperatorReconciler) mcpToNUMAResourceOperator(mcpObj clien
 	return requests
 }
 
+func (r *NUMAResourcesOperatorReconciler) getRelatedObjects(dsStatuses []nropv1.NamespacedName) []configv1.ObjectReference {
+	// 'Resource' should be in lowercase and plural
+	ret := []configv1.ObjectReference{
+		{
+			Resource: "namespaces",
+			Name:     r.Namespace,
+		},
+		{
+			Group:     "machineconfiguration.openshift.io",
+			Resource:  "kubeletconfigs",
+			Namespace: "",
+			Name:      "",
```

Comment:
Do we need to set namespace/name explicitely here? If so, let's add this data; if not, no need to explicit initialization with zero-value strings (= let's just omit these here and below)